### DISCUSSION TITLE
How can I change the font used in a quarto book?

### DISCUSSION THREAD

**Discussion by ddotta** [quarto-dev/quarto-cli#6030] [Q&A] [answered]
### Description

Hi,

In [my book](https://github.com/ddotta/cookbook-rpolars) I'd like to use a [special font](https://www.fontsquirrel.com/fonts/cascadia-code) to display the polars code correctly.  

Ideally I'd like to store the font ttf file in my github repository (for example in a font subdirectory) and then make a link.

I've tried this but quarto doesn't seem to like it:

```r
format:
  html:
    theme:
      light: cosmo
    code-fold: false
    css: custom.css
    mainfont: font/Cascadia.ttf # <- here's the problem
```

**Comment by mcanouil** [comment, marked as answer]
You need to import it in your (S)CSS file:

```css
@font-face {
  font-family: "MyFont";
  src: url("path/to/font.ttf");
}
```

Then you can use `mainfont: MyFont`.

**Reply by ritog** [reply]
Works great! Thanks for the response!

**Comment by ddotta** [comment]
Thanks @mcanouil  your solution works perfectly!
